rust/processor/src/processors/econia_processor.rs

```diff
@@ -782,7 +792,7 @@ impl ProcessorTrait for EconiaTransactionProcessor {
                                 .expect("Failed to parse MarketAccounts");
                             let map_field = data.get("map").expect("No map field");
                             market_account_handles.push(MarketAccountHandle {
-                                user: resource.address.clone(),
+                                user: strip_hex_number(resource.address.clone())?,
                                 handle: opt_value_to_string(map_field.get("handle"))?,
```

Comment:
`fn event_data_to_place_market_order_event` has `integrator` which is an address, as does `fn event_data_to_place_limit_order_event` and `fn event_data_to_place_swap_order_event`. Now that I look at it, `event_data_to_place_swap_order_event` also has `signing_account` which is another address.

Reply:
Just updated.